integration/test_resources/help/cloud-account_iac-generate

```diff
@@ -10,6 +10,7 @@ Available Commands:
   aws         Generate and/or execute Terraform code for AWS integration
   azure       Generate and/or execute Terraform code for Azure integration
   gcp         Generate and/or execute Terraform code for GCP integration
+  gke         Generate and/or execute Terraform code for GKE integration
```

Comment:
The command structure now seems a little odd. 
`aws`, `azure`, `gcp` relate to multiple modules/cloud accounts. In aws it can create either cfg or cloudtrail.

With this addition should each command  either  be 1:1 with a cloud account  or nested under cloud provider?

Reply:
+1 to this comment. Maybe @ipcrm can chime in here too but we need to follow a pattern and be consistent.